src/main/kotlin/org/rust/lang/core/types/re/ReUnderscore.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.types.re
+
+object ReUnderscore : Re() {
```

Comment:
Do we have this type of region in rustc? If not, I this it's processed on another level (like lowering) and we dont need it on type level too